base/src/iosMain/kotlin/DataTypeUtils.kt

```diff
@@ -0,0 +1,35 @@
+package com.splendo.kaluga.base
+
```

Comment:
Don't know how it's important by in almost all files missing copyright comment header

Reply:
Very important. It should be there automatically when you make a new file though, maybe something got broken. Which IDE are you using @Daeda88 ?

Reply:
Android Studio, but I probably disabled it a while ago